### PR TITLE
upgrade respec config

### PR DESCRIPTION
**Pull request by pfps**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-entailment/pull/24.html" title="Last updated on May 25, 2023, 5:27 PM UTC (bab43c3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-entailment/24/bdf712e...bab43c3.html" title="Last updated on May 25, 2023, 5:27 PM UTC (bab43c3)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -11,35 +11,31 @@
...
         ],
 
-        //doJsonLd:     true,
+        github:               "https://github.com/w3c/sparql-entailment",
+        group:                "rdf-star" ,
+        doJsonLd:     true,
+        wgPublicList:         "public-rdf-star-wg",
+//        implementationReportURI: "https://w3c.github.io/sparql12-entailment/reports/",
+//        errata:               "https://w3c.github.io/sparql12-entailment/errata/",
+//        localBiblio:          localBibliography,
```
Comment (pchampin) [inline spec/index.html:38]: I believe that this should not be commented out for the FPWD. This is what is causing 5 errors and 31 warnings in the editors draft.

Comment (pfps) [reply spec/index.html:38]: OK, I guess.

**Review by pfps — COMMENTED** [review]
OK, I guess

**Comment by pfps** [comment]
Wait?  What should be commented out?   semantic has the wgPublicList in it and it appears to be fine.

Maybe the problem is no xref?   But I don't think entailment ever had this, and it appeared to be fine.

**Comment by TallTed** [comment]
tl;dr: This wasn't about `wgPublicList`, it was about `localBiblio`

-----

The comment from @pchampin was about line 38, which you can confirm by clicking the "Add a suggestion" icon while looking at [the files area](https://github.com/w3c/sparql-entailment/pull/24/files) of this PR —

— as I did to get the below —
```suggestion
//        localBiblio:          localBibliography,
```
